src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage = () => {
+  const { tabId } = useParams();
+  const selectedTabId = tabId;
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={classNames({
+                'is-active': selectedTabId === tab.id,
+              })}
+            >
+              <Link
+                to={`/tabs/${tab.id}`}
+                data-cy="TabLink"
+                id={tab.id}
+              >
+                {tab.title}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.find(tab => tab.id === selectedTabId)
+          ? tabs.find(tab => tab.id === selectedTabId)?.content
+          : 'Please select a tab'}
```

Comment:
```suggestion
        {tabs.find(tab => tab.id === selectedTabId)?.content || 'Please select a tab'}
```
but better to move this to some `selectedTab` var